github/PullRequest.py

```diff
@@ -747,7 +747,7 @@ def merge(
         headers, data = self._requester.requestJsonAndCheck("PUT", f"{self.url}/merge", input=post_parameters)
         return github.PullRequestMergeStatus.PullRequestMergeStatus(self._requester, headers, data, completed=True)
 
-    def add_to_assignees(self, *assignees: github.NamedUser.NamedUser) -> None:
+    def add_to_assignees(self, *assignees: github.NamedUser.NamedUser | str) -> None:
         """
         :calls: `POST /repos/{owner}/{repo}/issues/{number}/assignees <https://docs.github.com/en/rest/reference/issues#assignees>`_
```

Comment:
can you also update this document link? this link is out of date

Reply:
Commited suggested change.